src/components/LayerList/HlsLayerForm.tsx

```diff
@@ -0,0 +1,78 @@
+import { useCallback, useEffect, useMemo, useState } from "react";
+import { LayerConfig, StacCollection } from "../../types";
+import { FormControl, FormLabel } from "@chakra-ui/react";
+// import DateRangeSlider from "../generic/DateRangeSlider";
+import { RangeDatepicker } from "chakra-dayzed-datepicker";
+import { durationToMs } from "../../utils";
+
+type Props = {
+  config: LayerConfig;
+  collection: StacCollection;
+  updateLayer: (config: LayerConfig) => void;
+}
+
+function HlsLayerForm({ config, collection, updateLayer }: Props) {
```

Comment:
I don't know how extensively this UI will get used, but its a nice UI so I could see us incorporating more datasets which require the same date picker so we should probably name this something more general like `LayerFormWithDatePicker` and the other to `LayerFormWithDateSlider`